signals/generators/ios/parameters.py

```diff
@@ -1,41 +1,118 @@
 """
-Creates list of Objective C specific method parameter names and types.
+Creates list of Objective-C or Swift specific method parameter names and types.
```

Comment:
Could we maybe make the `parameters.py` file either it's own module with a `base.py` and `objc_parameters.py` + swift one, etc or move them under the swift and objc template modules?